scripts/machine-setup.ps1

```diff
@@ -1,13 +1,12 @@
 # NOTE: DO NOT RUN THIS ON DEVELOPMENT MACHINES!
 # This is intended for setting up CI machines with the correct dependencies.
 
-$DotNetInstallDirectory = Join-Path -Path $HOME -ChildPath "dotnet"
 $ProgressPreference = 'SilentlyContinue'
 
 if (!(Get-Command python -ErrorAction SilentlyContinue ))
 {
   $PythonInstaller = New-TemporaryFile | Rename-Item -NewName { $_.Name + ".exe" } -PassThru
-  Invoke-WebRequest -Uri "https://www.python.org/ftp/python/3.8.3/python-3.8.3.exe" -OutFile $PythonInstaller
+  Invoke-WebRequest -Uri "https://www.python.org/ftp/python/3.10.4/python-3.10.4-amd64.exe" -OutFile $PythonInstaller
```

Comment:
We should extract this to a constant somewhere IMHO. Maybe we could even setup a build environment pipeline which rebuilds and pushes dockerfiles based on changes to them?